.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501
```

Comment:
Is there a reason the line length error is being ignored?

Reply:
I don't like it

Reply:
But seriously we should agree on these standards if we're going to change them, personally I think the line length limit is far too short and can discourage being explicit with function/variable names.

Reply:
You can configure it fairly easily: http://flake8.pycqa.org/en/2.6.0/config.html 
You can put it to what you want. The standard seems to be 160 max across SDC

Reply:
I'll try and implement that and see how many errors I get

Reply:
You can configure it to what is needed. I'm just not sure it should be ignored.